### DISCUSSION TITLE
Encryption in transit and at rest

### DISCUSSION THREAD

**Discussion by Bang-Kargo** [hasura/graphql-engine#8664] [General] [answered]
Hi,
Hope you can help me out. Is our data in our Hasura container in Azure encrypted in transit and at rest?

Thank you in advance from a non-tech person.

Br. Gita

**Comment by meetzaveri** [comment, marked as answer]
Hello, thanks for reaching out. 

Can we know that which platform you are using for your development ?
1. Hasura CE (open source version)?
2. Hasura Cloud (free tier, standard tier or pro) ?
3. Hasura Enterprise (VPC ) ?

**Comment by Bang-Kargo** [comment]
Hi Meet Zaveri
Thank you for coming back to me so quickly. We are using Hasura standard.

--
Med venlig hilsen / Best regards,
Gita Bang-Kargo
CEO & Chief Compliance Officer
***@***.***
Structuring regulatory complexity
Email: ***@***.******@***.***>
Office: +45 30 22 75 01
Komplio ApS
CVR: 41593261
Tagensvej 94, 1.tv, 2200 Copenhagen


Fra: Meet Zaveri ***@***.***>
Dato: mandag, 4. juli 2022 kl. 14.03
Til: hasura/graphql-engine ***@***.***>
Cc: Gita Bang-Kargo ***@***.***>, Author ***@***.***>
Emne: Re: [hasura/graphql-engine] Encryption in transit and at rest (Discussion #8664)

Hello, thanks for reaching out.

Can we know that which platform you are using for your development ?

  1.  Hasura CE (open source version)?
  2.  Hasura Cloud (free tier, standard tier or pro) ?
  3.  Hasura Enterprise (VPC ) ?

**Reply by meetzaveri** [reply]
Hello,

We would recommend creating a ticket under "Cloud Account Support" in "Help & support" section from your Hasura cloud dashboard. We are recommending this because it **involves private data discussion** and which we feel that if you create a ticket via "Cloud Account Support" then it would remain as a private discussion with our support team and your details regarding data wouldn't be exposed to the public as any github discussions are publicly accessible. 

Thanks!

**Comment by Bang-Kargo** [comment]
Thank you for your kind advice, I have filed a ticket.

Have a nice day.

--
Med venlig hilsen / Best regards,
Gita Bang-Kargo
CEO & Chief Compliance Officer
***@***.***
Structuring regulatory complexity
Email: ***@***.******@***.***>
Office: +45 30 22 75 01
Komplio ApS
CVR: 41593261
Tagensvej 94, 1.tv, 2200 Copenhagen


Fra: Meet Zaveri ***@***.***>
Dato: tirsdag, 5. juli 2022 kl. 14.34
Til: hasura/graphql-engine ***@***.***>
Cc: Gita Bang-Kargo ***@***.***>, Author ***@***.***>
Emne: Re: [hasura/graphql-engine] Encryption in transit and at rest (Discussion #8664)

Hello,

We would recommend creating a ticket under "Cloud Account Support" in "Help & support" section from your Hasura cloud dashboard. We are recommending this because it involves private data discussion and which we feel that if you create a ticket via "Cloud Account Support" then it would remain as a private discussion with our support team and your details regarding data wouldn't be exposed to the public as any github discussions are publicly accessible.

Thanks!
